### PR TITLE
Expands Jungle_syndicate.dmm

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Expands the Syndicate Bunker ruin out while maintaining its general loot and threat levels. The ruin now has a 30x30 foot print, area definitions, and a coherent powernet. I think it's pretty cool compared to the old version. 

The loot list has been expanded slightly but remains mostly unchanged. The ruin has:
Gear off of every syndicate you kill
2 18 force combat knives
1 C-20r
2 C-20r magazines
1 suspicious toolbox
1 Stechkin
2 Stechkin magazines
3 frag grenades
3 smoke grenades
Beer Nuke
1 pacman
1 combat glove
1 black market uplink
2 maid outfit boxes 
1 binocular
1 blood-red hardsuit

The turret count has been increased, although they remain not too hard to kill (so long as you are armed), and the mob count remains the same. Jerry now rules over the facility from the fort's command tower.

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
The ruin is less small, that's more cool. 

## Changelog

:cl:
add: Jungle_syndicate has been expanded from a tiny bunker to a small fortification, 
/:cl:

### CODE REVIEW

**Comment by Yellow-Mushroom** [comment]
Id avoid mixing reinforced walls + plastit windows, it creates some conflicting style

**Comment by Latentish** [comment]
image
Is that a fucking CARPETED SHRINKLET???

This is extremely funny and because it's a ruin I'll allow it. It must live.

File: code/game/area/areas/ruins/jungle.dm [hunk 1]
```diff
@@ -96,3 +96,33 @@
 	name = "cargo dump"
 	icon_state = "dk_yellow"
 
+// Syndicate Shitfort
+
+/area/ruin/jungle/shitfort
```
Comment (Latentish) [inline code/game/area/areas/ruins/jungle.dm]: Please rename your areas to not be "shitfort".

File: code/game/area/areas/ruins/jungle.dm [hunk 2]
```diff
@@ -96,3 +96,33 @@
 	name = "cargo dump"
 	icon_state = "dk_yellow"
 
+// Syndicate Shitfort
```
Comment (Latentish) [inline code/game/area/areas/ruins/jungle.dm]: This is quite an extreme amount of areas that could easily be replaced with only 1 unique area and some basic general areas.

**Review by Latentish — CHANGES_REQUESTED** [review]
Unless this is a magical smes, this won't power the ruin due to that pacman not starting as on.
image

**Comment by Erikafox** [comment]
> Unless this is a magical smes, this won't power the ruin due to that pacman not starting as on. image

Would setting it up start with charge also work?

File: code/game/area/areas/ruins/jungle.dm (hunk 2 above)
Comment (Erikafox) [reply code/game/area/areas/ruins/jungle.dm]: I gave Jerry all the shutters and stuff. The rest is the generic syndicate fort or away mission now.

File: code/game/area/areas/ruins/jungle.dm (hunk 1 above)
Comment (Erikafox) [reply code/game/area/areas/ruins/jungle.dm]: Done.

**Comment by Yellow-Mushroom** [comment]
One of the loot items is the black market uplink, however, when spawned in my own testing I failed to get any of the items to arrive no matter the method of transportation. 

You may want to switch this one out if you haven't tested it before, as while its a cool loot item, it may be broken

**Comment by Erikafox** [comment]
honestly I hadn't even processed that it did stuff. I think I'll just remove it for now. Unless someone has a replacement in a similar vein.

**Comment by Latentish** [comment]
> > Unless this is a magical smes, this won't power the ruin due to that pacman not starting as on. image
> 
> Would setting it up start with charge also work?

Some ships are docked for hours. I'd recommend a magical smes.

**Comment by SweetBlueSylveon** [comment]
You didn't write a datum for this. You need a datum for it to spawn.

**Comment by Latentish** [comment]
I wouldn't be against tming this while it's passing

**Comment by Latentish** [comment]
Pretty margeable to me.

**Review by Latentish — APPROVED** [review]

**Pull request merged**